src/librustc_trans/trans/debuginfo.rs

```diff
@@ -1156,12 +1156,42 @@ pub fn get_cleanup_debug_loc_for_ast_node<'a, 'tcx>(cx: &CrateContext<'a, 'tcx>,
         }
     }
 
-    NodeInfo {
+    NodeIdAndSpan {
         id: node_id,
         span: cleanup_span
     }
 }
 
+#[derive(Copy, Clone, PartialEq, Eq)]
+pub enum DebugLocation {
```

Comment:
If you keep the enum, make use of namespaced enum and dump the `DebugLoc` suffixes.